lms/djangoapps/course_wiki/plugins/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""Module with the course app configuration for the Wiki."""
+from typing import Dict, Optional, TYPE_CHECKING
+
+from django.conf import settings
+from django.utils.translation import ugettext_noop as _
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.course_apps.plugin import CourseApp
+
+if TYPE_CHECKING:
+    from django.contrib.auth import get_user_model
+    User = get_user_model()
+
+WIKI_ENABLED = settings.WIKI_ENABLED
```

Comment:
Nit: possible to use `SettingsToggle` here.

Reply:
As discussed this will be part of the cleanup work as this require a little discovery.